gapic/src/main/com/google/gapid/perfetto/views/CounterPanel.java

```diff
@@ -63,7 +63,10 @@ public CounterPanel copy() {
   @Override
   public String getTitle() {
     CounterInfo info = track.getCounter();
-    if (info.type == CounterInfo.Type.Gpu && "gpufreq".equals(info.name)) {
+    if(info.name.startsWith("power.rails")) {
+      return info.name.replace("power.rails.","").replace(".", "-");
+    }
+    else if (info.type == CounterInfo.Type.Gpu && "gpufreq".equals(info.name)) {
```

Comment:
Please follow Google Java style: space after `if` and no newline between closing brace and `else`/`else if`.

Reply:
My bad. Fixed it